src/tr/merlinscans/src/eu/kanade/tachiyomi/extension/tr/merlinscans/MerlinScans.kt

```diff
@@ -1,12 +1,431 @@
 package eu.kanade.tachiyomi.extension.tr.merlinscans
 
-import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class MerlinScans : MangaThemesia(
-    "Merlin Scans",
-    "https://merlinscans.com",
-    "tr",
-    dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale("tr")),
-)
+class MerlinScans : ParsedHttpSource() {
+
+    override val name = "MerlinScans"
+    override val baseUrl = "https://merlinscans.com"
+    override val lang = "tr"
+    override val supportsLatest = true
+
+    private val json = Json { ignoreUnknownKeys = true }
```

Comment:
`keiyoushi.utils` uses the [app instance](https://github.com/mihonapp/mihon/blob/c3b590cd3dfe70a5736411c6533913f2dadb17bc/app/src/main/java/eu/kanade/tachiyomi/di/AppModule.kt#L93) of `Json`, so you can remove this. If you need it for some reason use `private val json: Json by injectLazy()`